src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructorTest.java

```diff
@@ -63,9 +69,20 @@
 @RunWith(Theories.class)
 public class EventRecordConstructorTest {
 
-  @Rule public MockitoRule mocks = MockitoJUnit.rule();
-
-  @Mock Hive hiveMock;
+  @Rule
+  public MockitoRule mocks = MockitoJUnit.rule();
+
+  @Mock
+  Hive hiveMock;
+  @Mock
+  org.apache.hadoop.hive.metastore.api.Partition europePartition;
```

Comment:
Unless we really want to reuse these mocks, I would suggest moving them under the actual test to keep them focused:

```
public void preExecHook_shouldRetrievePartitions() throws Exception {
  org.apache.hadoop.hive.metastore.api.Partition europePartition = mock(org.apache.hadoop.hive.metastore.api.Partition.class);
```